test/extensions/filters/http/ext_authz/config_test.cc

```diff
@@ -3,95 +3,121 @@
 #include "envoy/extensions/filters/http/ext_authz/v3/ext_authz.pb.validate.h"
 #include "envoy/stats/scope.h"
 
+#include "source/common/grpc/async_client_manager_impl.h"
+#include "source/common/network/address_impl.h"
+#include "source/common/thread_local/thread_local_impl.h"
 #include "source/extensions/filters/http/ext_authz/config.h"
 
 #include "test/mocks/server/factory_context.h"
+#include "test/test_common/real_threads_test.h"
 #include "test/test_common/test_runtime.h"
 #include "test/test_common/utility.h"
 
+#include "absl/synchronization/barrier.h"
```

Comment:
are these 3 includes needed now? I think you get everything via RealThreadsTestBase.

Reply:
Thanks, removed.